qa/standalone/mon/mon-cluster-log.sh

```diff
@@ -62,7 +62,7 @@ function TEST_cluster_log_level() {
     ceph config set mon.a mon_cluster_log_level info
     ceph osd down 0
     TIMEOUT=20 wait_for_osd up 0 || return 1
-    grep -q "cluster [[]INF[]] osd.0.*boot" $dir/log
+    TIMEOUT=60 wait_for_string $dir/log "cluster [[]INF[]] osd.0.*boot" 
```

Comment:
while testing, encountered a couple failures in this test so added retries here as well.